config/piano.ts

```diff
@@ -4,8 +4,6 @@ export const TRACK_PLAYING_SPEED = 100;
 export const HORIZONTAL_GAP_BETWEEN_NOTES = 2;
 export const MS_PER_SECOND = 10;
 export const PIANO_HEIGHT = 210;
-export const ACCIDENTAL_KEY_COLOR = "#ffdc66";
-export const NATURAL_KEY_COLOR = "#42C9FF";
```

Comment:
I will still keep these here and import them in the file where I want to use them. It helps when defaults are present at a single place.

Reply:
moved the default values back under the constant named `DEFAULT_THEME`

Reply:
I previously changed the default colors to the ones that are part of the `TwitterPicker` default presets. Now that we use `ChromePicker` instead, which does not have the concept of presets, I changed the colors back.